packages/@react-spectrum/picker/intl/hu-HU.json

```diff
@@ -1,4 +1,4 @@
 {
-  "loading": "Betöltés folyamatban...",
+  "loading": "Betöltés folyamatban…",
```

Comment:
this changed to use an ellipsis, should all translations have this?

Reply:
It just needs to be consistent within a locale... hu-HU had mix of both, translator was correcting, but missed two places. I just updated PR for remaining two,